ArchiveScraping/targetSpeaker.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+
+
+def findSpeaker():
+    try:
+        df = pd.read_csv('bloomberg.tv.2.csv')
+    except:
+        print("file could not be found in directory")
+
+
+    # apply function below to remove any words matching company from description
+    speaker = df.apply(removeCompany, axis=1)
+
+    # Remove interview and undecessary symbols
+    removalWords = ['Bloomberg TV Interview','BTV Interview','LIVE <GO> Interview',':','-']
+    for x in removalWords:
+        speaker = speaker.str.replace(x,'')
+
+    # remove things of the nature (113223 MM)
+    df['Speaker'] = speaker.str.replace(r'\([a-z,A-Z,0-9, ]*\)', '')
+
+
+    #output file to csv
+    df.to_csv('bloombergAltered.tv.2.csv')
+
+
+def removeCompany(x):
+    try:
```

Comment:
Add a doc string giving a quick description of the function (and the other function above). See my other file for an example of the format.